mobile/src/main/java/org/openhab/habdroid/core/OpenHABVoiceService.java

```diff
@@ -34,68 +31,35 @@
  * This service handles voice commands and sends them to OpenHAB.
  * It will use the openHAB base URL if passed in the intent's extra.
  */
-public class OpenHABVoiceService extends Service implements ConnectionFactory.UpdateListener {
+public class OpenHABVoiceService extends IntentService {
```

Comment:
Wasn't it your argument to not use IntentService here? I can't really remember anymore, why, but if so, would you be so kind to explain to me, what've changed now? Just for my interest :)